### PR TITLE
Bump all examples and docs to v1alpha3

### PR DESCRIPTION
**Pull request by negz**
### Description of your changes

Updates the docs to match the below PRs.

https://github.com/crossplaneio/stack-aws/pull/58
https://github.com/crossplaneio/stack-gcp/pull/70
https://github.com/crossplaneio/stack-azure/pull/49

### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Ensured this PR contains a neat, self documenting set of commits.
- [ ] Updated any relevant [documentation], [examples], or [release notes].
- [ ] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.

[documentation]: https://github.com/crossplaneio/crossplane/tree/master/docs
[examples]: https://github.com/crossplaneio/crossplane/tree/master/cluster/examples
[release notes]: https://github.com/crossplaneio/crossplane/tree/master/PendingReleaseNotes.md
[`clusterrole.yaml`]: https://github.com/crossplaneio/crossplane/blob/master/cluster/charts/crossplane/templates/clusterrole.yaml

### CODE REVIEW

File: docs/stacks-guide-aws.md [hunk 1]
```diff
@@ -546,7 +546,7 @@ Below we inspect each of these resource classes in more details:
 
   ```yaml
   ---
-  apiVersion: database.aws.crossplane.io/v1alpha2
+  apiVersion: database.aws.crossplane.io/v1alpha3
```
[the remark this answers is not in the document]
Comment (negz) [reply docs/stacks-guide-aws.md]: Updated this example to the v1beta1 schema in a separate commit.

**Comment by soorena776** [comment]
@negz I'm doing a quick test on the three providers using this branch. will update when I'm done

File: docs/stacks-guide-aws.md [hunk 2]
```diff
@@ -573,7 +573,7 @@ Below we inspect each of these resource classes in more details:
 
   ```yaml
```
Comment (soorena776) [inline docs/stacks-guide-aws.md]: https://github.com/crossplaneio/crossplane/blob/master/cluster/examples/workloads/kubernetes/wordpress/aws/resource-classes/rds-class.yaml#L14
dbsubnetGroupNameRef -> dbSubnetGroupNameRef

Comment (negz) [reply docs/stacks-guide-aws.md]: How would you feel about opening another PR to address this? After some back and forth I decided to try keep this PR scoped to just bumping the version number. I think this issue needs fixing, but is not caused by or related to this PR.

Comment (soorena776) [reply docs/stacks-guide-aws.md]: sounds good, created https://github.com/crossplaneio/crossplane/pull/994/

**Pull request merged**